Examples/Image/TransferLearning/TransferLearning_distributed.py

```diff
@@ -0,0 +1,275 @@
+# Copyright (c) Microsoft. All rights reserved.
+
```

Comment:
Thanks for contributing!

Please add a test for this new script. See here (https://github.com/Microsoft/CNTK/blob/master/Tests/EndToEndTests/CNTKv2Python/Examples/transfer_learning_test.py) as an example.